src/main/java/Event.java

```diff
@@ -8,16 +8,17 @@ public Event(String description, String date, String time) {
         this.time = time;
     }
 
-    public Event(int mark, String description, String timeFrame) {
+    public Event(int mark, String description, String date, String time) {
         super(description, mark);
-        this.timeFrame = timeFrame;
+        this.date = date;
+        this.time = time;
     }
 
     public String getEvent() {
         return "[E]" + this.getTask() + "(at: " + this.date + ", " + this.time + ")\n";
     }
 
     public String getFormattedText() {
-        return "E:" + this.getMark() + ":" + this.getDescription() + ":" + this.timeFrame;
+        return "E>" + this.getMark() + ">" + this.getDescription() + ">" + this.date + ">" + this.time;
```

Comment:
I think we can use this.type instead of hardcoding "E" for the event. This can be done for the other tasks too:)